src/Aquifer.Public.API/Endpoints/Bibles/Books/List/LegacyEndpoint.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Aquifer.Public.API.Helpers;
+
+namespace Aquifer.Public.API.Endpoints.Bibles.Books.List;
+
+public class LegacyEndpoint : Endpoint
+{
+    public override void Configure()
+    {
+        // "/bible-books" is still supported for legacy compatibility but "/bibles/books" is the new hotness
+        Get("/bible-books");
+        Options(EndpointHelpers.UnauthenticatedServerCacheInSeconds(EndpointHelpers.OneHourInSeconds));
+
+        // we can eventually hide this endpoint from the docs altogether: https://fast-endpoints.com/docs/swagger-support#filtering-endpoints
```

Comment:
Can we just remove it actually?

Reply:
Sure.  I'll look into apply endpoint labels tomorrow.  I'm hoping we can just make a "no-docs" label or something like that and apply it with a single line of configuration after I wire things up.

Reply:
Done [here](https://github.com/BiblioNexusStudio/aquifer-server/pull/445/commits/f6621945b6df3c8565e0404f3648361e715887c9).  I'm pretty happy with this solution.